slack_cancel.gs

```diff
@@ -89,26 +83,26 @@ function cancel_su(rowindex){
 
   // reply to slack thread to confirm cancel (chat.postMessage method)
   var out_message = '<!channel> <@' + row.slackVolunteerID + '> is no longer available. Can anyone else volunteer? Type `/volunteer ' + row.uniqueid + '`.';
-  var options = {
-      method: "post",
-      contentType: 'application/json; charset=utf-8',
-      headers: {Authorization: 'Bearer ' + access_token},
-      payload: JSON.stringify({text: out_message,
-                               thread_ts: row.slackTS,
-                               reply_broadcast: true,
-                               channel: row.channelid})
-  };
-  var return_message = UrlFetchApp.fetch(webhook_chatPostMessage, options).getContentText(); // Send post request to Slack chat.postMessage API
+  var payload = JSON.stringify({
+    text: out_message,
+    thread_ts: row.slackTS,
+    reply_broadcast: true,
+    channel: row.channelid});
+
+  var return_message = postToSlack(payload, webhook_chatPostMessage);
 
   // if post request was unsuccesful, do not update tracking sheet and log error
   var return_params = JSON.parse(return_message);
   if (return_params.ok !== true){ // message was not successfully posted to channel
 
     // update log sheet
-    log_sheet.appendRow([new Date(), row.uniqueid,'admin','confirmCancel',return_message]);
+    log_sheet.appendRow(
```

Comment:
naive question: is the lineskip intentional?

Reply:
This is intentional - but not required. Tradition has it that lines should be no more than 80 characters long (https://softwareengineering.stackexchange.com/questions/148677). Splitting things like this makes it easier to read, in some opinions...